source/README.adoc

```diff
@@ -0,0 +1,142 @@
+
+:toc:
+:toc-placement!:
+= RabbitMQ Source
+
+The RabbitMQ source translates messages on a RabbitMQ exchange to CloudEvents
+over HTTP for use with Knative Eventing. The Source can bind to an existing
+exchange or create a new one, as needed.
+
+toc::[]
+
+
+== Published Events
+
+All messages received by the Source will be published with the following schema:
```

Comment:
```suggestion
All messages received by the source will be published with the following schema:
```

Use lowercase consistently for source IMHO

Reply:
Done (lowercasing)